exercises/gigasecond/.meta/gen.go

```diff
@@ -22,29 +22,32 @@ func main() {
 
 // The JSON structure we expect to be able to unmarshal into
 type js struct {
-	Add struct {
-		Description []string
+	Cases []struct {
+		Description string
 		Cases       []struct {
-			Input    string
-			Expected string
+			Description string
+			Input       string
+			Expected    string
 		}
 	}
 }
 
 // template applied to above data structure generates the Go test cases
 var tmpl = `package gigasecond
 
-// Source: {{.Ori}}
-{{if .Commit}}// Commit: {{.Commit}}
-{{end}}
-{{range .J.Add.Description}}// {{.}}
-{{end}}var addCases = []struct {
+{{.Header}}
+
+{{range .J.Cases}}// {{.Description}}
```

Comment:
I hope there won't ever be more than one element in that `.J.Cases`; that would cause odd things to happen (two `var addCases`).

We can deal with that whe nit happens in the future. Maybe it will never happen.